spec/cc/engine/rubocop_spec.rb

```diff
@@ -323,7 +323,7 @@ def includes_check?(output, cop_name)
       def includes_content_for?(output, cop_name)
         issue = issues(output).detect { |i| i["check_name"] =~ /#{cop_name}$/ }
 
-        issue["content"]["body"].present?
+        issue["content"] && issue["content"]["body"].present?
```

Comment:
Ahhh sorry. 
Fixed. Then made additional changed and squashed everything again. My bad. Updating.